docs/source/routing/about-v2.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: What's New in GraphOS Router v2.x
+subtitle: ""
+description: What's new in Apollo GraphOS Router version 2.x, including Long Term Release (LTS) status, key features, and how to upgrade
+---
+
+GraphOS Router v{products.router.version("lts_2_latest").version} is the latest Apollo runtime platform. This release introduces changes and enhancements that improve the router’s overall quality and establishes a strong foundation for new innovations, such as Apollo Connectors for REST. It also ships with the new Native Query Planner, optimized for efficiency and performance for high scale mission critical workloads. 
+
+## Release status of router v2.x
+
+GraphOS Router v2.x is an **Active** release with latest version {products.router.version("lts_2_latest").version}.
+
+| Current Status | Release Date | Latest Minor | Active Date    | Maintenance Date | End of Life Date |
+| -------------- | ------------ | ------------ | -------------- | ---------------- | ---------------- |
+| Active    | July 2024  | v{products.router.version("lts_2_latest").version}  |  February 2025 | -     | -     |
+
+As an [active](/graphos/reference/router-release-lifecycle#active) release:
+
+- Backwards-compatible minor releases are planned to ship monthly.
+- Minor releases can contain new features, bug fixes, and security patches.
+- Breaking changes may be required in extenuating circumstances.
+- [Support](https://www.apollographql.com/enterprise/support) is provided via paid support plans until end-of-life. 
+
+## What's new in router v2.x
+
+Below are the key highlights of the new and updated capabilities in router v2.x.
+
+> For detailed descriptions of updates in router v2.x, go to the [router 1.x to 2.x upgrade guide](/graphos/reference/upgrade/from-router-v1). 
+
+### Simplified integration and API orchestration of REST services using Apollo Connectors
+
+The router introduces **Apollo Connectors for REST** to enable the router to directly call REST APIs alongside traditional GraphQL subgraphs. It improves the interoperability of the router with existing API architectures without the need to write tedious API integration code.
```

Comment:
```suggestion
Apollo Router 2.0 introduces Apollo Connectors for REST, a declarative way to integrate REST services into supergraphs. By defining REST API integrations in GraphQL schema, you can orchestrate calls across your fleet of APIs and other services to produce a single response from a federated GraphQL API.
```

I received this edit from the Connectors team for my What's New doc. Applying it here

Reply:
```suggestion
The router v2.x introduces Apollo Connectors for REST, a declarative way to integrate REST services into supergraphs. By defining REST API integrations in GraphQL schema, you can orchestrate calls across your fleet of APIs and other services to produce a single response from a federated GraphQL API.
```